src/EditorFeatures/Core/Implementation/RenameTracking/RenameTrackingTaggerProvider.StateMachine.cs

```diff
@@ -308,12 +308,13 @@ internal int StoreCurrentTrackingSessionAndGenerateId()
                                 trackingSession.OriginalName,
                                 snapshotSpan.GetText());
 
-                            return new RenameTrackingCodeAction(
-                                document, title, refactorNotifyServices, undoHistoryRegistry);
+                            return (new RenameTrackingCodeAction(
+                                        document, title, refactorNotifyServices, undoHistoryRegistry),
+                                    snapshotSpan.Span.ToTextSpan());
```

Comment:
```suggestion
                            return (new RenameTrackingCodeAction(document, title, refactorNotifyServices, undoHistoryRegistry),
                                    snapshotSpan.Span.ToTextSpan());
```

Reply:
Added a note in https://github.com/dotnet/roslyn/issues/44855